dev_config/subscription.yml

```diff
@@ -2,3 +2,7 @@ workers: 5
 build-slaves: 4
 users: 15
 diskspace: 500MB
+moneybird:
+  # Eonics' contact_id
+  contact_id: '211786024202798994'
+  reference: 'Observ'
```

Comment:
since its yaml, drop the string quotation

Reply:
will do

Reply:
We need the string quotation for the `contact_id`, because otherwise it's interpreted as some kind of number. E.g. the `contact_id` of Eonics changed to `211786024202799000` for some reason